src/ActiveLogin.Identity.Swedish.AspNetCore/Validation/CoordinationNumberAttribute.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace ActiveLogin.Identity.Swedish.AspNetCore.Validation
+{
+    /// <summary>
+    /// Validates a Swedish Coordination Number using <see cref="CoordinationNumber"/>.
+    /// </summary>
+    public class CoordinationNumberAttribute : ValidationAttribute
+    {
+        /// <summary>Initializes a new instance of the <see cref="CoordinationNumberAttribute"></see> class.</summary>
+        public CoordinationNumberAttribute()
+            : base("{0} is not a valid Swedish Coordination Number. It should follow a valid pattern such as YYMMNN-XXXX, YYMMNN+XXXX or YYYYMMNNXXXX.")
```

Comment:
We have been using a slightly different way of representing Coordination numbers, see  the docs here:
https://github.com/ActiveLogin/ActiveLogin.Identity#swedish-coordination-number-1

Reply:
Thanks! I tried digging into Skatteverket's documentation, but the right representation was right under my nose.